MetroscopeModelingLibrary/Tests/Multifluid/HeatExchangers/Fogging_reverse.mo

```diff
@@ -4,17 +4,17 @@ model Fogging_reverse
   // Boundary conditions
   input Real P_fg_in(start = 1.1, min = 0, nominal = 10) "barA";
   input Utilities.Units.MassFlowRate Q_fg_in(start = 660) "kg/s";
-  input Utilities.Units.Temperature T_fg_in(start = 22.21) "degC";
+  input Real T_fg_in(start = 28.8)  "degC";
```

Comment:
I think the unit declaration is useful, it gives warning when you use the wrong unit in the sensor

Reply:
Yes but `Utilities.Units.Temperature` is in `K`, not in `degC`